docs/guides/writing-a-feature-app.md

```diff
@@ -226,12 +226,41 @@ const myFeatureAppDefinition = {
 };
 ```
 
+## Implementing a Feature App for an Integrator That Uses Web Components
+
+If the targeted integrator is using the [`@feature-hub/dom`](dom-api) package a
+Feature App needs to implement the `DomFeatureApp` interface the package
+defines. Since this interface is compatible with the `DomFeatureApp` interface
+defined by [`@feature-hub/react`](react-api) this Feature App will also be
+compatible with an integrator using React.
+
+A DOM Feature App allows fot the use of arbitrary frontend technologies such as
+Vue.js, Angular or React and is [placed on the web page using Web
+Components][placing-feature-apps-on-a-web-page-using-web-components]. The
+Feature App will automatically be enclosed in its own shadow DOM.
```

Comment:
I would add this sentence (from the React section above) as intro to the code snippet:

> Its definition's `create` method returns a Feature App instance with an `attachTo` method that accepts a DOM container element: